vite.dev-config.ts

```diff
@@ -1,16 +1,10 @@
-import react from '@vitejs/plugin-react'
-import { defineConfig } from 'vite'
+import { defineConfig, mergeConfig } from 'vite'
+import viteConfig from './vite.config'
 
 // https://vitejs.dev/config/
-export default defineConfig({
-  plugins: [react()],
+export default mergeConfig(viteConfig, defineConfig({
```

Comment:
This allows defining the dev config as modifications to the main config, avoiding repetition.